src/clojure_tictactoe/game/players.clj

```diff
@@ -9,3 +13,17 @@
 
 (defn player-swap []
   (swap! player-atom switch-player))
+
+(defn human-player-move [board]
+  (loop []
+    (let [move (input-getter/get-player-choice)]
+      (if (board/space-is-open? move board)
+          move
+          (do (println "Sorry, that looks taken, try again")
```

Comment:
This part looks like it should be moved to a namespace that knows about UI.

Reply:
Very true. I moved it, but I think I need help figuring out how to untangle board, players and input-getter.